Symfony/CS/ToolInfo.php

```diff
@@ -51,7 +51,14 @@ private static function getScriptDir()
             $script = $_SERVER['SCRIPT_NAME'];
 
             if (is_link($script)) {
-                $script = dirname($script).'/'.readlink($script);
+                $linkTarget = readlink($script);
+
+                // If the link target is relative to the link
+                if (false === realpath($linkTarget)) {
```

Comment:
Yes, it's true. You can read about it in the documentation: http://php.net/manual/en/function.realpath.php

```
Returns the canonicalized absolute pathname on success. The resulting path will have no symbolic link, '/./' or '/../' components.

realpath() returns FALSE on failure, e.g. if the file does not exist. 
```

Here the problem is that the **link target** doesn't exist if we check it from the **location that we execute the link**, but it exists from the **location of the link**. We have three locations:
* Location from we execute the link (```/var/www/myproject```)
* Location of the link (```/var/www/myproject/bin```)
* Location of the link target (```/var/www/myproject/vendor/fabpot/php-cs-fixer/php-cs-fixer```)